live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -1,5 +1,5 @@
 import styled from 'styled-components';
-import { max } from '../../../shared-ui/lib/responsive';
+import { max, min } from '../../../shared-ui/lib/responsive';
 import { colors } from '../../../shared-ui/style/colors';
 import { P } from '../../../shared-ui/style/typography';
 
```

Comment:
For the StyledWelcomeSectionContainer, can you make it max.tabletLg? It looks really cramped at those screensizes rn
<img width="885" alt="Screenshot 2023-01-23 at 12 36 35 AM" src="https://user-images.githubusercontent.com/59738880/213972327-f1f594d2-b3e8-4373-8d96-3bde2810b333.png">